src/common-components/RedirectLogistration.jsx

```diff
@@ -36,6 +37,10 @@ const RedirectLogistration = (props) => {
       // TODO: Do we still need this cookie?
       setCookie('van-504-returning-user', true);
       const registrationResult = { redirectUrl: finalRedirectUrl, success };
+      if (isRegistrationEmbedded) {
+        window.parent.postMessage({ message: 'redirection', redirectUrl: `${window.location.origin}${AUTHN_PROGRESSIVE_PROFILING}` }, '*');
+        return null;
+      }
```

Comment:
we can move above the line `const registrationResult = { redirectUrl: finalRedirectUrl, success };` because we dont need `registrationResult` in this case.